app/controllers/application_controller.rb

```diff
@@ -1,2 +1,27 @@
 class ApplicationController < ActionController::Base
+    before_action :configure_permitted_parameters, if: :devise_controller?
+    protect_from_forgery with: :exception
+    #helper_method :current_cart
+    helper_method :authorize_admin
```

Comment:
It was left over from the way I was going to do carts before. I left it commented out instead of removing it in case I came back to that way of doing it, but I've removed it now since I won't be using it.

Reply:
To be clear, I was referring to line 5:
`helper_method :authorize_admin`

It's completely fine to leave line 4 commented out.